web/src/lib/cms/previews/DashboardSnackbarsPreview.tsx

```diff
@@ -43,6 +43,16 @@ const DashboardSnackbarsPreview = (props: PreviewProps): ReactElement => {
           <Content>{config.dashboardDefaults.deferredOnboardingSnackbarBody}</Content>
         </SnackbarAlert>
 
+        <h2>Tax Registration Snackbar</h2>
+        <SnackbarAlert
+          variant="success"
+          isOpen={true}
+          close={(): void => {}}
+          heading={config.dashboardDefaults.taxRegistrationSnackbarHeading}
+        >
+          <Content>{config.dashboardDefaults.taxRegistrationSnackbarBody}</Content>
+        </SnackbarAlert>
+
```

Comment:
Just a thought and possible opportunity for a future refactor...

All of these snackbars are being generated this exact same way. It'd be cool to iterate over `config.dashboardDefaults` instead of manually making each one.